source/Patches/Roles/Lover.cs

```diff
@@ -9,24 +9,31 @@ namespace TownOfUs.Roles
 {
     public class Lover : Role
     {
-        public Lover(PlayerControl player, int num, bool loverImpostor) : base(player)
```

Comment:
The convention where `num == 2` for impostor seemed really brittle to me. So I changed it to always specify.